lib/evse_security/evse_security.cpp

```diff
@@ -955,25 +969,33 @@ bool EvseSecurity::update_certificate_links(LeafCertificateType certificate_type
 std::string EvseSecurity::get_verify_file(CaCertificateType certificate_type) {
     std::lock_guard<std::mutex> guard(EvseSecurity::security_mutex);
 
-    // Support bundle files, in case the certificates contain
-    // multiple entries (should be 3) as per the specification
-    X509CertificateBundle verify_file(this->ca_bundle_path_map.at(certificate_type), EncodingFormat::PEM);
+    try {
+        // Support bundle files, in case the certificates contain
+        // multiple entries (should be 3) as per the specification
+        X509CertificateBundle verify_file(this->ca_bundle_path_map.at(certificate_type), EncodingFormat::PEM);
 
-    EVLOG_debug << "Requesting certificate file: [" << conversions::ca_certificate_type_to_string(certificate_type)
-                << "] file:" << verify_file.get_path();
+        EVLOG_info << "Requesting certificate file: [" << conversions::ca_certificate_type_to_string(certificate_type)
```

Comment:
Is this intentionally a info message now?

Reply:
Yes, because sometimes we receive a security log, that does not print it's last action, and we are sometimes confused on "what actually was the last action executed by the lib".